app/javascript/packs/pattern_library.js

```diff
@@ -1,81 +1,90 @@
 import WebpackerReact from 'webpacker-react'
```

Comment:
what does this file do? (This is the first time I think I've encountered a PR that touches the pattern library)

Reply:
that's a good question, i'd actually like to know more details about this too! i believe Fed set it up initially, we could ask him for more details when he's back. broadly, i know it does setup for the pattern library, but i don't know the details about it.